README.md

```diff
@@ -4,13 +4,22 @@
 
 Below is the list of prerequisites to compile `komodo-ecosysboard` on your machine:
 
-- Golang 1.12.6 minimum
+- Golang 1.12.6 minimum (https://golang.org/doc/install)
 
 ## Getting Started
 
 These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
 See deployment for notes on how to deploy the project on a live system.
 
+### Dependancies
+```bash
```

Comment:
You dont need to use go get for the dependencies, simply use GO111MODULES=ON

Reply:
GO111MODULE=on go test ./... for example and it's should retrieve all the dependancies automatically

Reply:
Pull request are no in the main repository https://github.com/KomodoPlatform/komodo-ecosysboard/pull/2